src/icons/ui/Theme.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { forwardRef, Ref, SVGAttributes } from 'react';
+
+export const Theme = forwardRef(
+    (props: SVGAttributes<SVGSVGElement>, ref: Ref<SVGSVGElement>) => (
+        <svg
+            width="24"
+            height="24"
+            viewBox="0 0 24 24"
+            fill="none"
+            xmlns="http://www.w3.org/2000/svg"
+            ref={ref} {...props}
+                >
+            <g id="Icon/Dark/Theme">
+                <rect
+                    id="Rectangle"
+                    opacity="0.01"
+                    width="24"
+                    height="24"
+                />
+                <g id="photo_filter-24px">
```

Comment:
Is this tag necessary? Also, please remove the `id` attribute.

Reply:
Fixed